src/main/cpp/t08_cycleswap.cpp

```diff
@@ -19,5 +19,19 @@
 using namespace std;
 
 int t08_cycleswap() {
+    int n = 0;
+    cin >> n;
+    int arr[n];
+    for (int i=0; i<n; i++) {
+        cin >> arr[i]; }
 
+    int last = arr[n-1];
+    for (int i=n-1; i>0; i-=1) {
+        if (i == 0) arr[i] = last;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Opposite inner 'if' condition leads to a dead code block.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4432820)